server/app/routes/api/v1/organizations/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var router = require( 'express' )
+  .Router();
+var _ = require( 'lodash' );
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require( '../../../utils' );
+const {
```

Comment:
This is visually very different than how you have the other route indices. Be consistent 😃